src/qilib/data_set/mongo_data_set_io.py

```diff
@@ -150,7 +150,7 @@ def update_document(self, data: Dict[str, Any]) -> None:
 
     @staticmethod
     def encode_numpy_array(
-            array: Union[np.ndarray, DataArray]) -> Dict[str, Dict[str, Union[str, Tuple[int, ...], bytes]]]:
+            array: Union[np.ndarray, DataArray]) -> Dict[str, Union[str, Dict[str, Union[str, List[Any], bytes]]]]:
```

Comment:
What kind of return value is this? Can this be aliased for readability?

Reply:
I will look at it and try to come up with an alias